apps/dg/components/case_table/case_table_adapter.js

```diff
@@ -533,12 +539,13 @@ DG.CaseTableAdapter = SC.Object.extend( (function() // closure
             }).filter( function( iCase) {
               // We filter because, at least in FireFox, we can get to a row that is one beyond the end
               // which therefore has no case
-              return !SC.none( iCase);
+              return iCase && !iCase._isProtoCase;
             }),
             select: true,
             extend: false
           };
-      tContext.applyChange( tChange);
+      if (tChange.cases.length)
```

Comment:
This will bypass selection if the selection list has length 0.  I think this is the mechanism to clear existing selections is to send an empty list. I wonder if there is a place where the case table uses it in this way.